app.rb

```diff
@@ -6,14 +6,15 @@
 require_relative 'teacher'
 require_relative 'input'
 require_relative 'menu'
+require_relative 'storage'
 
 class App
   attr_accessor :books, :people, :rentals
 
   def initialize
     @books = []
     @rentals = []
-    @people = []
+    @people = Storage.read_data(Storage::PEOPLE_FILE_PATH, Person)
```

Comment:
👍 Good effort on reading stored data! 👍 However, there are a couple of adjustments needed to meet the project requirements:

- [ ]  As per the project requirement, on the startup of your app, you should load all data from files. Currently, it's giving an error, so please check the documentation and update your code to load all necessary data on startup without errors.

https://github.com/Obimbo07/school-library/assets/55840999/1e7f48aa-950f-40ef-85c7-0e8d2ce35636


- [ ]  In addition to loading people data, you also need to store and load data for `rentals` and `books`. Make sure to implement the necessary logic to store and load these data types along with people.

By addressing these points, you'll ensure that your app adheres to the project requirements. Keep up the good work! 🚀📦